src/js/mediaStore.js

```diff
@@ -0,0 +1,108 @@
+import { MediaUIEvents } from './constants.js';
+
+// Very similar to "MediaUIStates" except:
+export const stateFacade = {
```

Comment:
What do you mean by "facade" here?

Reply:
Referencing the design pattern name here https://en.wikipedia.org/wiki/Facade_pattern, though happy to call it whatever you like ("Names Are Hard™️")